src/plugins/intel_gpu/src/graph/include/convolution_inst.h

```diff
@@ -107,6 +107,18 @@ struct typed_program_node<convolution> : public typed_program_node_base<convolut
         return params;
     }
 
+    size_t hash() const override {
+        if (!seed) {
+            seed = parent::hash();
+
+            seed = hash_combine(seed, transposed);
+            seed = hash_combine(seed, groups);
+            seed = hash_combine(seed, deformable_groups);
+            seed = hash_combine(seed, deformable_mode);
```

Comment:
Same here for everything except `transposed`.

Reply:
I see, I will remove it.

Reply:
Removed it.